### PR TITLE
Div thm

### PR DESCRIPTION
**Pull request by nelong**

### CODE REVIEW

**Comment by mitchkeller** [comment]
I really will get around to giving this a look tomorrow morning. I have Stokes' done save the one exercise that I have mapped out in my head and want to make sure makes it in, so that will also get done tomorrow morning.

File: ptx/S_12_flux_int.ptx [hunk 1]
```diff
@@ -551,7 +551,8 @@
         </p>
       </task>
   </activity>
-
+ <p>
+  Look to Hughes Hallet for a note on Orientation of surfaces.
```
Comment (mitchkeller) [inline ptx/S_12_flux_int.ptx:555]: There's a missing `</p>` here that's causing this section not to compile properly. Is this a note of something to do during editing or something that you've got WIP on but haven't put in, @nelong?

File: ptx/S_12_Div_Thm.ptx [hunk 2]
```diff
@@ -16,91 +16,323 @@
...
+  <exercise>
+    <statement>
+      <p>
+        Explain why the flux of <m>\vF=\langle y,-z,x\rangle</m> through the unit sphere is zero. You should not compute any integrals as part of your explanation.
+      </p>
+    </statement>
+  </exercise>
+  <exercise>
+    <statement>
+      <p>Let <m>Q</m> be the volume given by <m> 0\geq z 3</m>, <m>1\geq r \geq 2</m>, and <m>0\geq \theta < 2 \pi</m>. Give an example of a vector field with component functions that are linear in <m>x</m>, <m>y</m>, and <m>z</m> such that the flux of the vector field through the boundary of <m>Q</m> is 17.</p>
```
Comment (mitchkeller) [inline ptx/S_12_Div_Thm.ptx:318]: @nelong There's a bit of a mess here in the definition of Q. I'm seeing some \geq that probably are supposed to be \leq? And then you can't use a naked `<` in XML, so if you need one, PTX supplies a \lt macro by default.

**Comment by mitchkeller** [comment]
@nelong The only two blocking issues for merging this have been flagged. If you push an update that fixes those, I think I can merge both this PR and mine and then shift to editing next week. (Thankfully I edit much faster than I write.)

File: ptx/S_12_flux_int.ptx (hunk 1 above)
Comment (nelong) [reply ptx/S_12_flux_int.ptx:555]: This is a something to do later note. We talked about needed to put some mention of orientability in this section (likely as an aside but maybe a section at the end). I think HughesHallet has a similar section to use as inspiration.

Comment (nelong) [reply ptx/S_12_flux_int.ptx:555]: I put in the </p>.

File: ptx/S_12_Div_Thm.ptx (hunk 2 above)
Comment (nelong) [reply ptx/S_12_Div_Thm.ptx:318]: I just missed this stuff. I think everything is fixed in these issues.

**Comment by nelong** [comment]
@mitchkeller I think these fixes are done. Let me know if you need anything else.

**Comment by mitchkeller** [comment]
@nelong Builds cleanly now. Thanks! I'm going to merge the PRs and perhaps spend some time this afternoon on getting the code to validate. Don't know that I have the brain power left this week to edit prose, but will work on that next week.

**Pull request merged**